packages/clerk-js/src/core/clerk.ts

```diff
@@ -465,6 +468,9 @@ export class Clerk implements ClerkInterface {
 
   public mountOrganizationProfile = (node: HTMLDivElement, props?: OrganizationProfileProps) => {
     this.assertComponentsReady(this.#componentControls);
+    if (noOrganizationExists(this) && this.#instanceType === 'development') {
+      return console.info(warnings.cannotRenderComponentWhenOrgDoesNotExist);
```

Comment:
🙃 i would suggest we split it into 2 different lines:
```
 console.info(warnings.cannotRenderComponentWhenOrgDoesNotExist);
 return;
```

Reply:
I kept that just for consistency from similar parts in the file.